src/config/consts.js

```diff
@@ -139,6 +139,13 @@ export const consts = {
     3: 3221225472
   },
 
+  MB_TO_BYTES: {
+    1: 1048576,
+    10: 10485760,
+    20: 20971520,
+    30: 31457280
+  },
```

Comment:
cool

Reply:
In the same way we have the `GB_TO_BYTES` object. Now we can use it for this kind of transformations.